kitsune/wiki/jinja2/wiki/includes/document_macros.html

```diff
@@ -411,4 +411,70 @@ <h3 class="card--title">
       </div>
     </section>
   {% endif %}
-{% endmacro %}
\ No newline at end of file
+{% endmacro %}
+
+{% macro kb_subtopic_tabs(request, product, topic, subtopics) -%}
+  <nav class="tabs">
+    <ul class="tabs--list subtopics">
+      <li class="tabs--item">
+        <a class="tabs--link is-active" href="#kb-subtopic-all" data-event-category="link click">
+          <span>{{ _('View All') }}</span>
+        </a>
+      </li>
+      {% for subtopic in subtopics %}
+        <li class="tabs--item">
+          <a class="tabs--link" id="kb-subtopic-{{ subtopic.title }}" 
+             href="#kb-subtopic-{{ subtopic.title }}" 
+             data-event-category="link click" 
+             data-event-action="subtopic sidebar" data-event-label="{{subtopic.title}}">
```

Comment:
What do you think about using the following instead?
- `data-event-category="device-migration-tabbed-panel"`
- `data-event-action="tab-click"`
- `data-event-label="{{subtopic.title}}"`